common.mk

```diff
@@ -568,7 +572,7 @@ FTPM_FLAGS ?= 						\
 .PHONY: ftpm
 ftpm:
 ifeq ($(MEASURED_BOOT_FTPM),y)
-ftpm: optee-os
+ftpm: optee-os-devkit
```

Comment:
This dependency looks good, but I don't see where/how the ftpm TA is linked into OP-TEE as an early TA. Perhaps this extra information should be omitted from the commit description.

Reply:
I was considering one building with `EARLY_TA_PATHS=/path/to/ftpm/build/bc50d971-d4c9-42c4-82cb-343fb7f37896.stripped.elf`.

Nevertheless, I'll remove the related information in the commit message.